src/lightning_app/MANIFEST.in

```diff
@@ -0,0 +1,7 @@
+include src/lightning_app/CHANGELOG.md
+include src/lightning_app/README.md
```

Comment:
TODO: just noticed that this needs to be rewritten to `src/lightning/app/CHANGELOG.md` when `PACKAGE_NAME=lightning`

This bug is also present in current master. So it could be fixed separately.

Reply:
Fixed in 86017c41a